source/server/admin/clusters_params.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "envoy/buffer/buffer.h"
+#include "envoy/http/codes.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Server {
+
+struct ClustersParams {
+  enum class Format {
+    Text,
+    Json,
+  };
+
+  Http::Code parse(absl::string_view url, Buffer::Instance& response);
```

Comment:
doxygen here please, e.g. it might not be obvious that response is only written if there's an error.